graphql/database/users/createUser.js

```diff
@@ -53,6 +57,24 @@ const createUser = async (
         }
       : null
   )
+
+  // Set default background image to a cat image if user is enabled for v4 beta
+  if (v4BetaEnabled) {
+    const backgroundCatImage = await getRandomBackgroundImage(
+      userContext,
+      BACKGROUND_IMAGE_CAT_CATEGORY
+    )
+    userInfo = {
+      ...userInfo,
+      backgroundImage: {
+        id: backgroundCatImage.id,
+        image: backgroundCatImage.image,
+        thumbnail: 'dummy',
```

Comment:
Sorry, I see [the thumbnail field is set to required in UserModel](https://github.com/gladly-team/tab/blob/master/graphql/database/users/UserModel.js#L168), but it actually no longer is. Go ahead and make it optional there and remove this thumbnail placeholder altogether.